src/components/TextArea.vue

```diff
@@ -11,6 +12,7 @@ import { Vue, Component, Prop } from "vue-property-decorator";
 @Component
 export default class TextArea extends Vue {
   @Prop({ default: "" }) readonly value!: string;
+  @Prop({ default: false }) readonly readonly!: boolean;
```

Comment:
I noticed that maybe better to rename as isReadonly. What do u think about it?

Reply:
I agree. It's better not to have variables same as reserved words
I haven't noticed. Good catch